torch/_torch_docs.py

```diff
@@ -4272,6 +4272,59 @@ def merge_dicts(*dicts):
     {out}
 """.format(**common_args))
 
+add_docstr(torch.xlogy,
+           r"""
+xlogy(x, y) -> Tensor
```

Comment:
xlogy(self, other) to be consistent with other PyTorch functions (we're not looking at consistency with NumPy for named arguments)